katsdptelstate/telescope_state.py

```diff
@@ -843,6 +890,8 @@ def add(self, key, value, ts=None, immutable=False, encoding=ENCODING_DEFAULT):
                                   "class method and thus cannot be used.")
          # check that we are not going to munge a class method
         key = _ensure_binary(key)
+        if self._INTERNAL_MARKER in key:
+            raise InvalidKeyError("Cannot set reserved keys")
```

Comment:
Might be worthing printing out the current _INTERNAL_MARKER to help the user understand why it is reserved.

Reply:
Will do.